code/examples/angular-cli/src/stories/addons/actions/addon-actions.stories.ts

```diff
@@ -1,46 +0,0 @@
-import { action } from '@storybook/addon-actions';
-import { Meta, StoryFn } from '@storybook/angular';
-import { Button } from '../../angular-demo';
-
-export default {
-  component: Button,
-  title: 'Addons/Actions',
-} as Meta;
-
-export const ComponentOutputWithEventEmitter: StoryFn = () => ({
-  props: {
-    text: 'Button 🥁',
-    onClick: action('On click'),
-  },
-});
-ComponentOutputWithEventEmitter.storyName = 'Component Output with EventEmitter';
-
-export const UseActionInMethod: StoryFn = () => ({
-  props: {
-    text: 'Button 🥁',
-    onClick: (e) => {
-      console.log(e);
-      e.preventDefault();
-      action('Action name')(e.target, 'Another arg');
```

Comment:
Do we have an example of using an action inline?

Reply:
an action inline? I'm not sure I understand..

Do you mean like this:
https://github.com/storybookjs/storybook/blob/802f349c498844900ea2a760f558334b4cd49991/code/addons/actions/template/stories/basics.stories.js#L19-L66

I can't do anything in a render function, in these stories, because that would make them framework specific.